lib/igniter/project/application.ex

```diff
@@ -90,14 +90,13 @@ defmodule Igniter.Project.Application do
          true <- Igniter.Code.List.list?(zipper),
          {:ok, zipper} <- Igniter.Code.Keyword.get_key(zipper, :mod) do
       case Igniter.Code.Common.expand_literal(zipper) do
-        {:ok, app_module} ->
+        {app_module, _} ->
           {:ok, app_module}
 
         :error ->
           try do
-            zipper.node
-            |> Code.eval_quoted()
-            |> elem(0)
+            {{module, _}, _} = Code.eval_quoted(zipper.node)
```

Comment:
This could return a `module` or `{app_module, opts}` as well, so need to handle that here.

Reply:
We also need to handle the possibility of it being `nil` (since an app isn't required), causing this function to return `nil`.